privacyidea/static/components/config/controllers/eventController.js

```diff
@@ -65,6 +65,17 @@ myApp.controller("eventController", ["$scope", "$stateParams", "$state",
         );
     };
 
+    $scope.confirmDeleteEvent = function (eventid) {
```

Comment:
This would be superflous

Reply:
fixed in 9c208ab4212d0e7fd5a0133faf6722115b3a40c9.